internal/slicer/slicer_test.go

```diff
@@ -22,13 +23,14 @@ var (
 )
 
 type slicerTest struct {
-	summary string
-	arch    string
-	release map[string]string
-	slices  []setup.SliceKey
-	hackopt func(c *C, opts *slicer.RunOptions)
-	result  map[string]string
-	error   string
+	summary      string
+	arch         string
+	release      map[string]string
+	slices       []setup.SliceKey
+	hackopt      func(c *C, opts *slicer.RunOptions)
+	fsResult     map[string]string
+	reportResult map[string]string
```

Comment:
[Note for reviewer]: Please not that all the results of the report do not conform to the business logic yet. Namely:
* Parent directories of {text} files are reported even though they should not.
* We do not track removed directories or changes done in Starlark.

These two will come as separate PRs that need their own discussion on how to implement each of the issues. Basically the tests right now keep track of all the `Create` calls, that's it.

Reply:
Sounds fine. Can we please have this note added to the implementation as a TODO comment?  This is a relevant divergence in the algorithm, so should be there until the issue is addressed.